test/spec/modules/rtbhouseBidAdapter_spec.js

```diff
@@ -19,7 +19,11 @@ describe('RTBHouseAdapter', () => {
         'region': 'prebid-eu'
       },
       'adUnitCode': 'adunit-code',
-      'sizes': [[300, 250], [300, 600]],
+      'mediaTypes': {
```

Comment:
Wolałbym zostawić wsteczną kompatybilność - tj żeby nasz adapterek działał zarówno dla Prebid 3 jaki wcześniejszych [czyli zarówno z tablicy sizes jak i obiektu mediaTypes

Reply:
Patrząc po innych PRach, w szczegółności tych zaakceptowanych wszyscy grzecznie zmieniają ze starego na nowy. Więc trochę nie widzę sensu (i szans na apprve) w zachowywaniu wstecznej kompatybilności.

Reply:
Wsteczna kompatybilność wydaje się być prosta, więc też raczej byłbym za zachowaniem, może oszczędzić problemów - mniejsza szansa, że będziemy tym partnerem, który będzie musiał tłumaczyć wydawcom jak powinni skonfigurować nową wersję ;)

Reply:
Problemem nie jest to czy jest trudna czy prosta, a to czy pozwolą nam taką zmiankę dodać. Mogę zmienić i zobaczymy co się stanie ;)

Reply:
Niżej jest sprawdzenie wstecznej kompatybilności :)